facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -91,53 +79,72 @@
  * <br><br>
  * terasology -help    or    terasology /?
- * <br><br>
+ *
  */
 
-public final class Terasology {
-
-    private static final String[] PRINT_USAGE_FLAGS = {"--help", "-help", "/help", "-h", "/h", "-?", "/?"};
-    private static final String USE_CURRENT_DIR_AS_HOME = "-homedir";
-    private static final String USE_SPECIFIED_DIR_AS_HOME = "-homedir=";
-    private static final String START_HEADLESS = "-headless";
-    private static final String LOAD_LAST_GAME = "-loadlastgame";
-    private static final String CREATE_LAST_GAME = "-createlastgame";
-    private static final String NO_CRASH_REPORT = "-noCrashReport";
-    private static final String NO_SAVE_GAMES = "-noSaveGames";
-    private static final String PERMISSIVE_SECURITY = "-permissiveSecurity";
-    private static final String NO_SOUND = "-noSound";
-    private static final String NO_SPLASH = "-noSplash";
-    private static final String SERVER_PORT = "-serverPort=";
-    private static final String OVERRIDE_DEFAULT_CONFIG = "-overrideDefaultConfig=";
-    private static final Logger logger = LoggerFactory.getLogger(Terasology.class);
+@CommandLine.Command(name = "terasology")
+public final class Terasology implements Callable<Integer> {
+
+    @Option(names = {"--help", "-help", "/help", "-h", "/h", "-?", "/?"}, usageHelp = true, description = "show help")
+    private boolean helpRequested;
+
+    @Option(names = "--headless", description = "Start headless")
+    private boolean isHeadless;
 
+    @Option(names = "--crash-report", defaultValue = "true", negatable = true, description = "Enable crash reporting")
+    private boolean crashReportEnabled;
 
-    private static boolean isHeadless;
-    private static boolean crashReportEnabled = true;
-    private static boolean soundEnabled = true;
-    private static boolean splashEnabled = true;
-    private static boolean loadLastGame;
-    private static boolean createLastGame;
+    @Option(names = "--sound", defaultValue = "true", negatable = true, description = "Enable sound")
+    private boolean soundEnabled;
 
+    @Option(names = "--splash", defaultValue = "true", negatable = true, description = "Enable splash screen")
+    private boolean splashEnabled;
+
+    @Option(names = "--load-last-game", description = "Load the latest game on startup")
+    private boolean loadLastGame;
+
+    @Option(names = "--create-last-game", description = "Recreates the world of the latest game with a new save file on startup")
+    private boolean createLastGame;
+
+    @Option(names = "--permissive-security")
+    private boolean permissiveSecurity;
+
+    @Option(names = "--save-games", defaultValue = "true", negatable = true, description = "Enable new save games")
+    private boolean saveGamesEnabled;
+
+    @Option(names = "--server-port", description = "Change the server port")
+    private Integer serverPort;
+
+    @Option(names = "--config", description = "Override default config")
```

Comment:
I changed this from `overrideDefaultConfig` to simply `config` because I believed it was overriding the location of the default configuration file, and I thought that a `config` pointing to a file was clear enough for that.

but now I think this maybe specifies an **additional** config file that contains overrides. Not something used **instead** of the default config. If that's true, it should get its "override" name back.